src/logging.js

```diff
@@ -1,41 +1,49 @@
+// @flow
+
 /**
  * Logging setup for our application.
  */
-const stream = require("stream");
-const expressWinston = require("express-winston");
-const winston = require("winston");
+import stream from "stream";
+import expressWinston from "express-winston";
+import winston from "winston";
+
+import {LoggingWinston as StackdriverTransport} from "@google-cloud/logging-winston";
 
-const StackdriverTransport = require("@google-cloud/logging-winston")
-    .LoggingWinston;
-const args = require("./arguments.js");
+import type {NpmLogLevels, Format} from "winston";
+import args from "./arguments.js";
+
+import type {Info, Logger, LogLevel} from "./types.js";
 
 /**
  * This is how the log message gets formatted.
  * We're adding the durationMs from the profiler commands in winston so
  * that we can read them :)
  */
-const prodFormatter = ({message, durationMs}) =>
+const prodFormatter = ({message, durationMs}: Info): string =>
     `${message} ${(durationMs && `(${durationMs}ms)`) || ""}`;
 
 /**
  * Our dev format adds the log level.
  */
-const devFormatter = (info) => `${info.level}: ${prodFormatter(info)}`;
+const devFormatter = (info: Info): string =>
+    `${info.level}: ${prodFormatter(info)}`;
 
 function getFormatters(json, isDev) {
-    const formatters = [
+    const formatters: Array<Format> = [
         winston.format.splat(), // Allows for %s style substitutions
     ];
 
     if (json) {
         formatters.push(winston.format.json());
         if (isDev) {
-            formatters.push(winston.format.prettyPrint({colorize: true}));
```

Comment:
Oh - does this option not do anything? Or are the Flow types wrong?

Reply:
Hmm, great question. I need to follow-up on that as I don't remember that I checked.

Reply:
According to the docs, this defaults to `false` so I'm going to put this back in and make a change to the flow types.

https://github.com/winstonjs/logform#prettyprint